packages/www/components/StreamsTable/index.tsx

```diff
@@ -242,15 +247,28 @@ export default ({ userId, id }: { userId: string; id: string }) => {
         <Button
           variant="primarySmall"
           aria-label="Delete Stream button"
-          disabled={!selectedStream}
-          onClick={() => selectedStream && setDeleteModal(true)}
+          disabled={!selectedStream.length}
+          onClick={() => selectedStream.length && setDeleteModal(true)}
         >
           Delete
         </Button>
       </Box>
       <Table sx={{ gridTemplateColumns: "auto auto auto auto auto auto" }}>
         <TableRow variant={TableRowVariant.Header}>
-          <Box></Box>
+          <Flex
+            sx={{ alignItems: "baseline", pt: "8px" }}
```

Comment:
Can we remove the padding top so the checkbox is centered?
<img width="630" alt="Screen Shot 2020-11-20 at 2 36 03 PM" src="https://user-images.githubusercontent.com/555740/99842522-fb0f1080-2b3d-11eb-8a57-fceed0cd1aa5.png">

Reply:
fixed